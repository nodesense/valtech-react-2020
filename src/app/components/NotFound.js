import React from 'react';

const NotFound = () => {
    return (
        <React.Fragment>
            <h2>Page not found</h2>
        </React.Fragment>
    )
}

export default NotFound;