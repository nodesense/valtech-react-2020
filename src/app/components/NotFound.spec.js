import renderer from 'react-test-renderer';
import React from 'react';

import NotFound from './NotFound';

describe("not found test suite", () => {
    it("not found default values", () => {
        const tree = renderer
            .create(<NotFound />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})