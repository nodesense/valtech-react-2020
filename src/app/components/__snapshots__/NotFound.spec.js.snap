// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found test suite not found default values 1`] = `
<h2>
  Page not found
</h2>
`;
